templates/python/.github/workflows/publish.yaml

```diff
@@ -0,0 +1,30 @@
+name: Publish
+# Template adapted from https://github.com/bakdata/ci-templates/tree/main/.github/workflows#python-poetry-release
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
I think we should change the default trigger to only publish from version tags

```suggestion
on:
   push:
     tags: ["v[0-9]+.[0-9]+.[0-9]+"]
```

Reply:
Makes sense, yes. done